Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -704,37 +706,20 @@ void CGPathAddRoundedRect(
     FAIL_FAST_IF_FAILED(path->AddGeometryToPathWithTransformation(rectangleGeometry.Get(), transform));
 }
 
-int _CGPathPointCountForElementType(CGPathElementType type) {
-    int pointCount = 0;
-
-    switch (type) {
-        case kCGPathElementMoveToPoint:
-        case kCGPathElementAddLineToPoint:
-            pointCount = 1;
-            break;
-        case kCGPathElementAddQuadCurveToPoint:
-            pointCount = 2;
-            break;
-        case kCGPathElementAddCurveToPoint:
-            pointCount = 3;
-            break;
-        case kCGPathElementCloseSubpath:
-            pointCount = 0;
-            break;
-    }
-    return pointCount;
-}
-
 /**
- @Status Stub
+@Status Caveat
+@Notes Quadratic Bezier Curves are simplified into Cubic Bezier curves. Control point approximation for arcs differs from reference
+platform. TODO 1419 : Fix figure logic in D2D to eliminate extra start point callbacks.
 */
 void CGPathApply(CGPathRef path, void* info, CGPathApplierFunction function) {
```

Comment:
It's perfectly fine. We're doing nothing with it, just handing it off to the CGPathApplier function. If they use their own NULL incorrectly then that's their fault. :P

---
In reply to: [91437902](https://github.com/Microsoft/WinObjC/pull/1507#discussion_r91437902) [](ancestors = 91437902)